aten/src/THC/THCTensorRandom.cuh

```diff
@@ -286,6 +288,10 @@ sampleMultinomialWithReplacement(curandStateMtgp32* state,
   // what, all block threads must participate in the curand_uniform
   // call to update the generator state.
```

Comment:
This comment is no longer valid (w/o mtgp, individual threads can participate in rng call)